client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/task-helper.ts

```diff
@@ -130,12 +135,12 @@ describe( 'Task Helpers', () => {
 				} );
 				const tasks = [ task ];
 				const site = defaultSiteDetails;
-				const newsletterFlow = NEWSLETTER_FLOW;
-				expect( filterDomainUpsellTask( newsletterFlow, tasks, site ) ).toBe( tasks );
+				const videopressFlow = VIDEOPRESS_FLOW;
+				expect( filterDomainUpsellTask( videopressFlow, tasks, site ) ).toBe( tasks );
```

Comment:
These seems like the same test/check as the test above - it seems to be testing that the upsell task is still in the list of enhanced tasks. Don't we want to remove that upsell for videopress flows?